src/EntityListener/Interfaces/EntityFileInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace AppBundle\EntityListener\Interfaces;
```

Comment:
maybe move this class in `AppBundle\Entity` ?

Reply:
in \Entity it's only mapped entities no ?

Reply:
No, there are also some interfaces implemented by these entity classes.